src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -100,24 +127,32 @@ public Uri Endpoint
 
         set
         {
+            Guard.ThrowIfNull(value);
```

Comment:
we should add the changelog for this.

Reply:
Added: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5434/commits/82951f87ad69292d96f2a45124e244698c431e17